app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import { ListingDetailItem, t } from "@bloom-housing/ui-components"
+import { RailsListing } from "../listings/SharedHelpers"
+import { renderInlineWithInnerHTML } from "../../util/languageUtil"
+import { stripMostTags } from "../../util/filterUtil"
+
+export interface ListingDetailsAdditionalInformationProps {
+  listing: RailsListing
```

Comment:
For now, passing in the listing object to each listing details component. We should discuss advantages/disadvantages to passing in only necessary props rather than the entire object